wire/msgmixdcnet_test.go

```diff
@@ -85,7 +85,7 @@ func TestMsgMixDCNetWire(t *testing.T) {
 	expected = append(expected, repeat(0x91, 20)...)
 	expected = append(expected, repeat(0x92, 20)...)
 	expected = append(expected, repeat(0x93, 20)...)
-	// Four seen DCs (repeating 32 bytes of 0x94, 0x95, 0x96, 0x97)
+	// Four seen SRs (repeating 32 bytes of 0x94, 0x95, 0x96, 0x97)
```

Comment:
found this typo as i was copying code for testing expected serialization of SR messages.  Both DC and FP reference back to the SRs.